app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -92,11 +92,13 @@
     private boolean servicesShown = false;
     private ImageView serviceArrow;
 
-    private static final int ITEM_ID_SUBSCRIPTIONS = - 1;
-    private static final int ITEM_ID_FEED = - 2;
-    private static final int ITEM_ID_BOOKMARKS = - 3;
-    private static final int ITEM_ID_DOWNLOADS = - 4;
-    private static final int ITEM_ID_HISTORY = - 5;
+
+    private static final int ITEM_ID_HOME = - 1;
+    private static final int ITEM_ID_SUBSCRIPTIONS = - 2;
+    private static final int ITEM_ID_FEED = - 3;
+    private static final int ITEM_ID_BOOKMARKS = - 4;
+    private static final int ITEM_ID_DOWNLOADS = - 5;
+    private static final int ITEM_ID_HISTORY = - 6;
```

Comment:
Why were these values like that?
```suggestion
    private static final int ITEM_ID_HOME = -1;
    private static final int ITEM_ID_SUBSCRIPTIONS = -2;
    private static final int ITEM_ID_FEED = -3;
    private static final int ITEM_ID_BOOKMARKS = -4;
    private static final int ITEM_ID_DOWNLOADS = -5;
    private static final int ITEM_ID_HISTORY = -6;
```

Reply:
Are you talking about the negative values? That I ain't sure about. It's already there so I let it stay.

Reply:
Yeah, I know ;-) I'd just make them normal anyway

Reply:
Done